### PR TITLE
Fix OWD-30110

### PR DESCRIPTION
**Pull request by willyaranda**
This makes PNS to send messages through websocket for a UDP client that is still connected (resetting the timeout)

### CODE REVIEW

File: src/common/DataStore.js [hunk 1]
```diff
@@ -479,7 +505,8 @@ var DataStore = function() {
                     // Also, we only care if the data.no is just one. We should not allow
                     // register more than one node for each appToken… but that's another
                     // story…
-                    if (!data || !Array.isArray(data.no) || data.no.length !== 1 || !data.no[0]) {
+                    if (!data || !Array.isArray(data.no) || data.no.length !==
+                        1 || !data.no[0]) {
```
Comment (frsela) [inline src/common/DataStore.js]: Better split after the || - will be clearer

File: src/ns_ua/connectors/ConnectorUDP.js [hunk 2]
```diff
@@ -21,7 +21,8 @@ ConnectorUDP.prototype = {
     },
 
     getServer: function() {
-        return 'UDP';
+        //return 'UDP';
```
Comment (frsela) [inline src/ns_ua/connectors/ConnectorUDP.js]: No dead code

File: src/ns_ua/connectors/ConnectorUDP.js [hunk 3]
```diff
@@ -21,7 +21,8 @@ ConnectorUDP.prototype = {
...
+        return process.serverId;
```
Comment (frsela) [inline src/ns_ua/connectors/ConnectorUDP.js:24]: When UDP is returned now?

File: src/common/DataStore.js (hunk 1 above)
Comment (willyaranda) [inline src/common/DataStore.js]: This is made automatically by `fixgjslint` which I think it is real crap

**Comment by willyaranda** [comment]
Fixed.

**Comment by frsela** [comment]
\o/
r+

File: src/common/DataStore.js (hunk 1 above)
Comment (frsela) [inline src/common/DataStore.js]: Should we change the linter? :p

**Pull request merged**